### PR TITLE
FEATURE: Underline condition query with fallbacks

### PR DESCRIPTION
**Pull request by Nukecola2005**

### CODE REVIEW

**Review by ASchmidt1024 — DISMISSED** [review]
🚀  Nice!

File: src/css/style.css [hunk 1]
```diff
@@ -718,3 +718,22 @@ img[style="float: right;"], img[style="float:right;"], img[style="float: right"]
...
+    text-align: var(--h-border-after-text-align, center) !important;
+    position: relative;
+  }
+  h1::after {
+    background: var(--h-border-after-color, var(--h1-color, var(--color, black))); 
+    content: '';
+    display: block; 
+    height: var(--h-border-after-height, auto); 
+    position: absolute;
+    left: var(--h-border-after-left-spacing, auto);
```
Comment (Weedshaker) [inline src/css/style.css:732]: --h-border-after-left

File: src/css/style.css [hunk 2]
```diff
@@ -718,3 +718,22 @@ img[style="float: right;"], img[style="float:right;"], img[style="float: right"]
...
+    transform: translateX(var(--h-border-after-translateX, 0));
```
Comment (Weedshaker) [inline src/css/style.css:733]: --h-border-after-transform

File: src/css/style.css [hunk 3]
```diff
@@ -718,3 +718,22 @@ img[style="float: right;"], img[style="float:right;"], img[style="float: right"]
     height: var(--spacer-four-height-mobile, var(--spacer-four-height, calc(4 * var(--content-spacing-mobile, var(--content-spacing)))));
   }
 }
+@container style(--h-border-after: true) {
+  h1{
+    text-align: var(--h-border-after-text-align, center) !important;
+    position: relative;
+  }
+  h1::after {
+    background: var(--h-border-after-color, var(--h1-color, var(--color, black)));
```
Comment (Weedshaker) [inline src/css/style.css:727]: --h-border-after-background-color

**Review by Weedshaker — CHANGES_REQUESTED** [review]
Try to always stay true to the prop name inside the variable name.

**Pull request closed without merging**